sbncode/CAFMaker/FillReco.cxx

```diff
@@ -161,7 +149,27 @@ namespace caf
           srshower.wirePitch_plane2 = plane.WirePitch()/cosgamma;
       }
     }
+  }
+
+  void FillShowerRazzle(const art::Ptr<sbn::MVAPID> razzle,
+      caf::SRShower& srshower,
+      bool allowEmpty)
+  {
+    srshower.razzle.electronScore = razzle->mMVAScoreMap.at(11);
+    srshower.razzle.photonScore = razzle->mMVAScoreMap.at(22);
+    srshower.razzle.otherScore = razzle->mMVAScoreMap.at(0);
 
+    srshower.razzle.pdg = razzle->BestPDG();
+    srshower.razzle.bestScore = razzle->BestScore();
+  }
+
+
+  void FillShowerCosmicCylinder(const std::vector<art::Ptr<float> >& cosmicCylinderVec,
+                      caf::SRShower& srshower)
+  {
+      if (cosmicCylinderVec.size() != 1)
+        return;
+      srshower.cosmicCylinder = *cosmicCylinderVec.front();
```

Comment:
`cosmicCylinder` is just a float? What is it, a radius? Can we give it a more evocative name?